util/logutil/log_test.go

```diff
@@ -149,3 +150,37 @@ func (s *testLogSuite) TestSlowQueryLoggerKeepOrder(c *C) {
 	expectMsg = fmt.Sprintf("log_test.go:%v: [warning] slow-query a=a b=b c=c d=d e=e f=f\n", line+1)
 	c.Assert(s.buf.String(), Equals, expectMsg)
 }
+
+func (s *testLogSuite) TestSlowQueryZapLogger(c *C) {
+	fileName := "slow_query"
+	conf := NewLogConfig("info", DefaultLogFormat, fileName, EmptyFileLogConfig, false)
+	err := InitZapLogger(conf)
+	c.Assert(err, IsNil)
+	defer os.Remove(fileName)
+
+	rets := []string{
+		`[INFO] [log_test.go:167] ["info message"] ["str key"=val]`,
```

Comment:
This code is vulnerable, maybe use regexp to check the result pattern is better?

Reply:
Done.